### ISSUE TITLE
"Процеси" course - "Kickoff" lesson - HTML validation - "Attribute aria-role not allowed on element div at this point" error is present

### ISSUE THREAD

**Issue by DmitriiNasonov** [scholokov/qax-portal-2#1835]
**Steps of reproducing**
1. Open https://validator.w3.org/
2. Check the [Kickoff lesson page](https://www.qax-camp.com.ua/courses/processes/lessons/kickoff/)

**Actual Result**
"Attribute aria-role not allowed on element [div](https://html.spec.whatwg.org/multipage/#the-div-element) at this point" error is present

**Expected Result**
"Attribute aria-role not allowed on element [div](https://html.spec.whatwg.org/multipage/#the-div-element) at this point" error is missing

**Environments**
Windows 10, Chrome 120.0.6099.110 (64 bit)
![Showing results for https___www qax-camp com ua_courses_processes_ - Nu Html Checker - Google Chrome 19 12 2023 17_40_35](https://github.com/scholokov/qax-portal-2/assets/95171672/98d68cf0-d0fa-4eb7-8ef7-465854483b57)

**Comment by DmitriiNasonov** [comment]
#1063
